subset/password/brute_test/src/main/java/MACHandler.java

```diff
@@ -0,0 +1,32 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class MACHandler {
+
+  public static String runShellCommand(String command) throws IOException {
+    Process p = Runtime.getRuntime().exec(command);
+    BufferedReader input = new BufferedReader(new InputStreamReader(p.getInputStream()));
+    String line;
+    while ((line = input.readLine()) != null) {
+      if (!line.trim().equals("")) {
+        line = line.substring(0);
+        String macAddress = getMACAddressFromCommand(line);
+        if (macAddress.isEmpty() == false) {
+          return macAddress;
+        }
+      }
+    }
+    return null;
+  }
+
+  public static String getMACAddressFromCommand(String macLine) {
+    String lineContents[] = macLine.split("   ");
+    for (String string : lineContents) {
+      if (string.trim().length() == 17) {

```

Comment:
wtf is 17?

we need to find a better way to do this, even if it's something like CONSTANT_STRING.lenght(), but then I don't understand why you wouldn't be comparing against that rather than just the length.